ehitchhiking-ui/src/app/shared/modals/profile-modal/profile-modal.component.sass

```diff
@@ -60,3 +60,7 @@
             margin-bottom: $default-offset
         .add-car-form
             padding: $default-offset
+
+    .no-data-wrapper
+        height: 20rem
```

Comment:
you should not override app-no-data styles, pls set `position: relative` to the container inside which should be displayed  app-no-data component